### PR TITLE
Fix positioning logic

### PR DESCRIPTION
**Pull request by benoneal**

### CODE REVIEW

File: source/client.js [hunk 1]
```diff
@@ -6,17 +6,17 @@ import RadialList from './components/RadialList'
...
+  <div>1</div>,
+  <div>2</div>,
+  <div>3</div>,
+  <div>4</div>,
+  <div>5</div>,
+  <div>6</div>,
+  <div>7</div>,
+  <div>8</div>,
+  <div>9</div>,
+  <div>10</div>
```
Comment (HowlingEverett) [inline source/client.js:19]: "An unnecessary id property killed my family. Now I'm out for revenge."

File: source/components/RadialList/styles.js [hunk 2]
```diff
@@ -3,6 +3,7 @@ export default {
     display: 'block',
     position: 'relative',
     margin: 0,
-    padding: 0
+    padding: 0,
+    transition: 'all 500ms ease-in-out'
```
Comment (HowlingEverett) [inline source/components/RadialList/styles.js:7]: Any reason for the transition all? If you need a couple more properties you can chain them easily.

Comment (benoneal) [reply source/components/RadialList/styles.js:7]: just easier, and not harmful

Comment (HowlingEverett) [reply source/components/RadialList/styles.js:7]: Eh :P

**Pull request merged**